src/app/action-items/action-items.component.html

```diff
@@ -18,16 +18,17 @@
                 <div class="flexRow bodyHeader" role="columnheader">Completed</div>
                 <div class="flexRow bodyHeader" role="columnheader">Due Date</div>
                 <div class="flexRow bodyHeader" role="columnheader">Days Left</div>
+                <div class="flexRow bodyHeader" role="columnheader"></div>
             </div>
         </div>
-        <div *ngIf="isloadingActionItems" class="spinner">
+        <div *ngIf="isLoadingActionItems" class="spinner">
             <i class="fa fa-spinner fa-spin"></i>
         </div>
-        <div class="tBody" *ngIf="!isloadingActionItems">
-            <div class="flexTable row singleItem" role="rowgroup" *ngFor="let t of dataSource">
+        <div class="tBody" *ngIf="!isLoadingActionItems">
+            <div *ngFor="let t of dataSource" id="{{ t.id }}" [routerLink]="['/items']" [queryParams]="{ id: t.id }" routerLinkActive="router-link-active" class="flexTable row singleItem" role="rowgroup">
```

Comment:
It would be better instead of `t` name this variable something like `actionItem`